src/main/java/com/pankaj/tictactoegame/models/Grid.java

```diff
@@ -0,0 +1,83 @@
+package com.pankaj.tictactoegame.models;
+
+import java.util.List;
+
+public class Grid {
+	public Grid() {
+		cells = generateCells();
+	}
+
+	private Cell[][] generateCells() {
+		Cell[][] cells = new Cell[4][4];
+		for(int i=1; i<=3 ; i++) {
+			for(int j=1; j<=3; j++) {
+				Cell cell = new Cell();
+				cells[i][j]=cell;
+			}
+		}	
+		return cells;
+	}
+
+	private Cell[][] cells;
+
+	public boolean playerMove(int row, int col, Player player) {
+		Cell cell = cells[row][col];
+		if (cell.getIsOccupied()) {
+			System.out.println("Invalid Move!!!");
```

Comment:
I suggest to write all print statement(in real world output of application) in the application/service layer and not in the model